### PR TITLE
docs(self-managed): external-elasticsearch configuration

### PR DESCRIPTION
**Pull request by hamza-m-masood**
## Description

related to: 
epic: https://github.com/camunda/distribution/issues/179
product-hub issue: https://github.com/camunda/product-hub/issues/1946
I am creating docs on how to setup external elasticsearch and aws managed opensearch.
## When should this change go live?

- [x] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by hamza-m-masood** [comment]
I will create a similar file like this for opensearch.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+id: using-existing-elasticsearch
+title: "Using existing Elasticsearch"
+description: "Learn how to use an existing elasticsearch instance in Camunda 8 Self-Managed deployment."
+---
+
+Camunda 8 Self-Managed has two different types of applications: Camunda applications (Operate, Optimize, Tasklist, etc.) and non-Camunda applications (such as Keycloak and Elasticsearch). For more details, review the [architecture](../../../platform-architecture/overview.md) documentation for more information on the different types of applications.
```
Comment (hisImminence) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:7]: > Camunda 8 Self-Managed has two different types of applications: Camunda applications (Operate, Optimize, Tasklist, etc.) and non-Camunda applications (such as Keycloak and Elasticsearch)

i would use the terms used in the architecture slide (Camunda Component and Dependency Components).

Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: done

**Comment by christinaausley** [comment]
Is this in `draft` because there are additional changes coming? If not, happy to review 👍

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:1]: In the body of the docs, it should always be `Elasticsearch` (not `elasticsearch`).

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+id: using-existing-elasticsearch
+title: "Using existing Elasticsearch"
+description: "Learn how to use an existing elasticsearch instance in Camunda 8 Self-Managed deployment."
+---
+
+Camunda 8 Self-Managed has two different types of components: Camunda components (Operate, Optimize, Tasklist, etc.) and non-Camunda, dependency components (such as Keycloak and Elasticsearch). For more details, review the [architecture](../../../platform-architecture/overview.md) documentation for more information on the different types of applications.
+
+This guide steps through using an existing elasticsearch instance. By default, [Helm chart deployment](../deploy.md) creates a new elassticsearch instance, but it's possible to use an existing elasticsearch instance either inside the same Kubernetes cluster or outside of it.
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:9]: This paragraph missing cohesion. I'd suggest rewording it (probably by staring with `By default ...`.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+## Preparation
+
+### Connecting to self managed elasticsearch
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:13]: self-managed

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+<!-- You must be aware of the username and password needed to connect to your Elasticsearch cluster. -->
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:15]: Any reason for this comment?

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+<!-- You must be aware of the hostname of the elasticsearch cluster. -->
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:16]: Any reason for those 2 comments?

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+You must be aware of the following information relating to your self managed elasticsearch cluster:
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:18]: Probably `aware` is not the best word here.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+1. protocol, host, port
+2. username and password
+
+Both `http` and `https` connections are possible when connecting to self managed elasticsearch by modifying `global.elasticsearch.protocol`
+
+If you are using self signed certificates and are accepting only `https` requests in your elasticsearch cluster then you must create a `.jks` file from your elasticsearch certificate file using the `keystore` tool. Then you must create a kubernetes secret from the `.jks` file before installing Camunda. For example, this is how you would create the `.jks` file and kubernetes secret from your elasticsearch certificate file:
+
+```yaml
+keytool -import -alias elasticsearch -keystore externaldb.jks -storetype jks -file <name of elasticsearch crt file> -storepass changeit -noprompt
+kubectl  create secret -n <namespace> generic <secret name> --from-file=externaldb.jks
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:29]: In our docs, we agreed to use example names instead of placeholders like `<namespace>` and `<secret name>`.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+### Connecting to es-cloud
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:32]: > Connecting to Elastic Cloud

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+Since es-cloud does not use self signed certificates, all you need is the following information:
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:34]: Let's use the actual name here like `Elastic Cloud`

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 9 above)
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:32]: > Connecting to managed Elasticsearch

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 10 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:34]: Also, clarify here that should work with any managed Elasticsearch and here we use `Elastic Cloud` as an example.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+1. protocol, host, port
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:36]: Let's capitalize the words here.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+## Preparation
+
+### Connecting to self managed elasticsearch
+
+<!-- You must be aware of the username and password needed to connect to your Elasticsearch cluster. -->
+<!-- You must be aware of the hostname of the elasticsearch cluster. -->
+
+You must be aware of the following information relating to your self managed elasticsearch cluster:
+
+1. protocol, host, port
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:20]: Let's capitalize the words here.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+### Connecting to es-cloud
+
+Since es-cloud does not use self signed certificates, all you need is the following information:
+
+1. protocol, host, port
+2. username and password
+
+You do not need to create a secret including the `.jks` file before installing camunda like the previous section.
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:39]: Why?

Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:39]: Let's explain why in 1 line.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md:1]: The official name is `OpenSearch`, let's use it like that.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+
+Camunda 8 Self-Managed has two different types of components: Camunda components (Operate, Optimize, Tasklist, etc.) and non-Camunda, dependency components (such as Keycloak and Elasticsearch). For more details, review the [architecture](../../../platform-architecture/overview.md) documentation for more information on the different types of applications.
+
+This guide steps through using an existing AWS managed opensearch instance. By default, [Helm chart deployment](../deploy.md) creates a new elassticsearch instance, but it's possible to use AWS managed opensearch instead.
+
+## Preparation
+
+### Authentication
+
+There are two layers of permissions with opensearch. IAM and Opensearch internal. If you would like to connect to Opensearch using IAM roles for service accounts (IRSA) then please also refer to this [page](../platforms/amazon-eks/irsa.md)
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md:15]: `AWS IAM`

Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md:15]: > please also refer to this [page](...)

For better readability, we should use a full title for the link instead of just `page`.

Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md:15]: Also, let's link to the `#opensearch` section directly since the page has other irrelevant content.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+
+Otherwise, if you wish to connect to AWS managed opensearch with basic auth the you can follow the example given below.
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md:17]: It's missing what the values should be used when IRSA is used.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+
+You can use the same values provided above and not include the `global.elasticsearch.tls` section since the tls section is only needed to specify self signed certificates.
+
+## Next Steps
+
+Then, use the custom values file to [deploy Camunda 8](../deploy.md) as usual.
+
+```sh
+helm install camunda camunda/camunda-platform -f existing-elasticsearch-values.yaml
+```
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:87]: @hamza-m-masood After reading the whole page, I'd suggest focusing on the use case instead of on the instructions so the user doesn't need to jump between sections.

So instead of 

```
## Preparation

### Connecting to self managed elasticsearch

### Connecting to es-cloud

## Values file

### Connecting to self managed elasticsearch with self signed certificates

### Connecting to es-cloud

## Next Steps
```

Make it:

```
## Connecting to existing Elasticsearch without a certificate

## Connecting to existing Elasticsearch with a self-signed certificate

## Connecting to existing Elasticsearch with a publicly trusted certificate

## Next Steps
```

**Review by aabouzaid — CHANGES_REQUESTED** [review]
@hamza-m-masood 
I've left some major and minor comments.


Also, please don't forget to add the migration steps since the new syntax has a breaking change (also reflect that in the other guides in the docs).

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 13 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:39]: Not sure what you mean.
Why the protocol, host, port is needed?
OR why the jks file is not needed?
I added further explanation on why the jks file is not needed.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 2 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:1]: done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 4 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: changed

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 6 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: removed

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 7 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: reworded

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 8 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:26]: removed placeholders

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 10 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: changed naming and clarified.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 9 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 11 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 12 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 3 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:9]: Maybe the lack in cohesion is caused by the first paragraph. I removed the first paragraph and reworded the second paragraph.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 17 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:109]: changed the layout

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 13 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: I removed this sentence with the new structure.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 2 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:1]: Apart from the inside code block, there are still some places not captured.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 12 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: The same for the next line `username and password`.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 10 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: We still need to keep the `Elastic Cloud` (as an example for managed Elasticsearch) sine it's a common keyword.

Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: In that case, I can replace this title: 
`## Connecting to existing Elasticsearch with a publicly trusted certificate`
with this title:
`## Connecting to Elastic Cloud`

**Comment by christinaausley** [comment]
Let me know when you are ready for final review 👍

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 2 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:1]: Sorry. i was messagin around with the `git commit --amend` command which took away some of my work. 😄 
should be good now.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 12 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md (hunk 14 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md:1]: done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md (hunk 15 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md]: - changed to AWS IAM
- added full title
- linked to the specific opensearch section

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md (hunk 16 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md]: I think it's better to add that in the IRSA doc itself. I'll do that.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+id: using-existing-elasticsearch
+title: "Using Existing Elasticsearch"
+description: "Learn how to use an existing Elasticsearch instance in Camunda 8 Self-Managed deployment."
+---
+
+By default, the [Helm chart deployment](../deploy.md) creates a new elassticsearch instance, but it's possible to use an existing Elasticsearch instance either inside the same Kubernetes cluster or outside of it. This guide steps through using an existing Elasticsearch instance.
+
+## Connecting to existing Elasticsearch without a certificate
+
+Connecting to Elasticsearch through `http` is possible by modifying `global.elasticsearch.protocol`
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:11]: @hamza-m-masood This needs a rewording since it's the default behavior, the user doesn't need to set or modify anything.
Also, the key now is `global.elasticsearch.url.protocol`.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+
+The following information must be known relating to the self-managed Elasticsearch cluster:
+
+1. Protocol, Host, Port
+2. Username and Password
+
+The Camunda 8 self-managed Helm chart can then be configured as follows:
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md:18]: `Camunda 8 self-managed` -> `Camunda 8 Self-Managed` (everywhere in the docs).

File: docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md [hunk 20]
```diff
@@ -181,6 +184,29 @@ New:
...
 
+#### Enabling External Elasticsearch
+
+It is possible to use external Elastichsearch. For more information on how to setup external Elasticsearch, please refer to the [Using Existing Elasticsearch](./guides/using-existing-elasticsearch.md) setup guide.
+
+##### Elasticsearch - values file
+
+The `global.elasticsearch.disableExporter` field has been deprecated in favour of `global.elasticsearch.enabled`. When `global.elasticsearch.enabled` is set to false, all configurations for Elasticsearch in all components are removed.
+
+The `global.elasticsearch.url` field has changed. If you are using the default values.yaml and have not configured the URL, then no change is required. On the other hand, if the URL value is used, then instead of specifying a single URL, you must now explicitly specify the Protocol, Host, and Port seperately like so:
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md:195]: We also need to mention the other changed keys `global.elasticsearch.protocol`, `global.elasticsearch.host`, and `global.elasticsearch.port`.

**Review by aabouzaid — APPROVED** [review]
LGTM, just minor changes are needed :tada:

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md (hunk 18 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md]: reworded

File: docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md (hunk 20 above)
Comment (hamza-m-masood) [reply docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md:195]: done

**Comment by hamza-m-masood** [comment]
@christinaausley I'm ready for your review now. 👍

**Review by aabouzaid — APPROVED** [review]

**Comment by christinaausley** [comment]
@hamza-m-masood I believe you'll just need to add these files to the `optimize_sidebars.js` and `sidebars.js` files.

**Review by aabouzaid — DISMISSED** [review]

**Review by christinaausley — CHANGES_REQUESTED** [review]
Add to sidebars 👍

**Comment by hamza-m-masood** [comment]
> Add to sidebars 👍

Apologies for the messy commits. I reverted the accidental merge.
I added in my files.

**Review by christinaausley — DISMISSED** [review]

**Review by christinaausley — DISMISSED** [review]

**Review by christinaausley — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]
